src/spark-connect/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[dependencies]
```

Comment:
Lets put this under `src/generated/spark-connect`

Reply:
ab6c56ba978e84a37b65077b368bd408446e25c8